common/lib/xmodule/xmodule/modulestore/xml_exporter.py

```diff
@@ -41,7 +41,7 @@ def export_to_xml(modulestore, contentstore, course_key, root_dir, course_dir):
 
     with modulestore.bulk_operations(course_key):
 
-        course = modulestore.get_course(course_key, depth=None)  # None means infinite
+        course = modulestore.get_course(course_key, depth=None, lazy=False)  # depth=None means infinite
```

Comment:
For code readability, I would suggest changing the name of the `lazy` parameter to be more explicit, like `lazy_loading_of_definitions` or adding a comment here.